packages/repository-tests/src/crud-test-suite.ts

```diff
@@ -32,6 +32,7 @@ export function crudRepositoryTestSuite(
     idType: 'string',
     freeFormProperties: true,
     emptyValue: undefined,
+    supportsTransactions: false,
```

Comment:
New feature flags should default to `true`. That way we include the new tests in all downstream repos (e.g. `repository-mysql`), which makes it much easier to discover a database/connector that does not support the new feature.